src/server/pps/server/worker_rc.go

```diff
@@ -88,6 +88,22 @@ func getPachctlSecretVolumeAndMount(secret string) (v1.Volume, v1.VolumeMount) {
 		}
 }
 
+// getTLSCertSecretVolumeAndMount returns a Volume and VolumeMount object
+// configured for the pach-tls secret to be stored in pipeline side-cars.
+func getTLSCertSecretVolumeAndMount(secret, mountPath string) (v1.Volume, v1.VolumeMount) {
+	return v1.Volume{
+			Name: secret,
```

Comment:
Is name of the volume something we control? Maybe call it something like `<secret>-volume` to distinguish it from the actual secret name?

Reply:
it is something that we control. We could do that, but to be consistent with other volumes, mounts we're creating here, I think just using the secret name makes sense